style.css

```diff
@@ -5,19 +5,28 @@
 }
 
 html {
-    font-size: 16px;
+    font-size: 11px;
+}
```

Comment:
gdy jest jeden wynik wyszukiwania to kefelek wyglada brzydko

![image](https://user-images.githubusercontent.com/17024927/145158392-87ae6a94-ad49-4d85-8f6a-ee078852ea48.png)

Reply:
Poprawione.